### DISCUSSION TITLE
Feature Request: Support for React 19

### DISCUSSION THREAD

**Discussion by dgtllion** [ueberdosis/tiptap#5816] [Feature Requests]
### Description

is there a plan to make @tiptap/react work with React 19?
When i try to upgrade to React 19 with pnpm i get warning that package is not supporting React 19.

```
├─┬ @tiptap/react 2.9.1
  ├── ✕ unmet peer react@"^17.0.0 || ^18.0.0": found 19.0.0-rc-66855b96-20241106
  └── ✕ unmet peer react-dom@"^17.0.0 || ^18.0.0": found 19.0.0-rc-66855b96-20241106

```

**Comment by nperez0111** [comment]
Already merged, just not released: https://github.com/ueberdosis/tiptap/pull/5807

**Reply by the3flower** [reply]
@nperez0111 Do you have a release date for Tiptap/React supporting React 19? :pray:

**Reply by nperez0111** [reply]
its already released

**Reply by the3flower** [reply]
Thank you very much
